cmd/flowrunner/main.go

```diff
@@ -43,7 +45,7 @@ const contactJSON = `{
 }
 `
 
-const usage = `usage: flowrunner [flags] <assets.json> <flow_uuid>`
+const usage = `usage: flowrunner [flags] <assets.json> [flow_uuid]`
```

Comment:
this is an unrelated change to make `flowrunner` easier to use by defaulting to first flow it finds in an assets file